zh/option/series/tree.md

```diff
@@ -204,6 +204,8 @@ center: [115.97, 29.71]
 
 {{ use: partial-focus-blur-scope(
     isTree = true
+    hasRelative = true
+    version = "5.3.3"
```

Comment:
It seems the `version` variable is not necessary?

Reply:
You mean only `hasRelative` here?
The reason I added `version`, it's other `series` who might have same feature in the future version.